package.json

```diff
@@ -21,40 +21,49 @@
         "XMLWriter": "dimagi/XMLWriter#master",
         "bootstrap": "3.4.1",
         "codemirror": "5.30.0",
-        "fuse.js": "~2.7.4",
-        "jquery": "~3.5.1",
+        "fuse.js": "2.7.4",
+        "jquery": "3.5.1",
```

Comment:
Why switch to pinning to specific versions rather than using approximate version specifiers?

One disadvantage of this is that every point release must be explicitly updated rather than happening automatically on the next build. Is there a new workflow to work around that?

Reply:
hmm, i guess we can keep the approximate ones `~`. at least with `jquery` i wanted it to be exactly the same as HQ's unless both are explicitly updated. for `fuse.js` I don't have a great answer

Reply:
Pinning `jquery` to the same version used by HQ seems reasonable.